backbone.blazer.js

```diff
@@ -58,6 +58,7 @@ Backbone.Blazer.Router = Backbone.Router.extend({
         var router = this;
 
         route.trigger('before:execute', routeData);
+        router.trigger('before:execute', routeData);
```

Comment:
i have to wire that stuff in to route data. probably can do that in a follow-up PR. working on something like that here: https://github.com/Betterment/backbone.blazer/compare/enhance-route-data